src/components/CategoryFilter.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from "react";
+import { css, StyleSheet } from "aphrodite";
+import { colors } from "src/styles";
+import { i18n } from "src/locale";
+import { Category } from "src/types";
+import { useNotification } from "src/hooks";
+import { Categories } from "./Categories";
+import { CircleIconButton, IconButtonsNames } from "./CircleIconButton";
+
+interface Props {
+  handleFilter: (filter: Category) => void;
+  activeCategory: Category;
+}
+
+export const CategoryFilter: FC<Props> = ({ handleFilter, activeCategory }) => {
+  const { handleNotification } = useNotification();
+
+  const handleClick = (_filter: Category) => {
+    console.log(_filter);
+    handleFilter(_filter);
+    if (_filter === activeCategory) {
+      handleFilter("none");
```

Comment:
smth is incorrect here. To many setStates

also do smth like that for checks -> `const  isTheSameCategory = _filter === activeCategory`
and then -> `handleFilter(isTheSameCategory ? 'none' : _filter )` and same for `text` in the notification